app/models/user.rb

```diff
@@ -11,4 +11,25 @@ def self.authenticate(email, password)
     user = User.find_for_authentication(email: email)
     user&.valid_password?(password) ? user : nil
   end
+
+  def create_access_token(client_app_id)
```

Comment:
Dies it works? I think `@id` value will be `nil` here

Reply:
:O 
Yes, totally true! Fixed!